docs/pages/installation/kinD.md

```diff
@@ -19,10 +19,46 @@ curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.8.1/kind-$(uname)-amd64
 chmod +x ./kind
 mv ./kind /some-dir-in-your-PATH/kind
 ```
+If you are running Ubuntu on WSL2, use `Docker Ubuntu` distro to install `Docker`. 
+
+### Create cluster using KinD
+
+In order to let you successfully access Meshery server from your localhost, you need to follow the specific instructions
```

Comment:
Hey @tingkai-zhang , just a small change I would suggest if you would incorporate.
```suggestion
In order to successfully build the meshery server on your local machine, you will need to follow these specific instructions
```

Reply:
Thanks @kushthedude I think that way makes it more clear to the readers.